taskvine/src/bindings/python3/ndcctools/taskvine/futures.py

```diff
@@ -45,16 +49,34 @@ def __init__(self, port=9123, batch_type="local", manager=None, manager_host_por
         else: 
             self.factory = None
 
-    def submit(self, fn, *args, **kwargs):
-        if isinstance(fn, FutureTask):
-            self.manager.submit(fn)
-            return fn._future
-        future_task = FutureTask(self.manager, False, fn, *args, **kwargs)
-        self.manager.submit(future_task)
-        return future_task._future
+    def submit(self, task):
```

Comment:
Here we want to keep the definition of submit to be (`(self, fn, *args, *kwargs)`). If the fn argument is actually of type `FuturePythonTask` or `FutureFunctionCall` then we just submit the task. if fn is a function we create a `FuturePythonTask` 

This is so that our executor conforms to [this](https://docs.python.org/3/library/concurrent.futures.html) so that we can use this in other applications that use futures.